src/config.js

```diff
@@ -612,6 +594,26 @@ export function newConfig() {
     }
   }
 
+  function mergeConfig(obj) {
+    if (!utils.isPlainObject(obj)) {
+      utils.logError('mergeConfig input must be an object');
+      return;
+    }
+
+    const mergedConfig = Object.keys(obj).reduce((accum, data) => {
+      const prevConf = _getConfig(data)[data] || {};
+      accum[data] = utils.mergeDeep(prevConf, obj[data]);
+      return accum;
+    }, {});
+
+    setConfig({ ...mergedConfig });
```

Comment:
spread operators are not supported in IE, unfortunately. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
I think the plan is to transpile the spread operator for some time.

Reply:
Is the consensus that I should replace the spread operator or is it being transpiled? @patmmccann